### PR TITLE
Web UI code cleanup and refactoring

### PR DESCRIPTION
**Pull request by rdmark**
- Split up source files that were getting too large
- Clean up of the status messages shown in the Web UI
- Resolved a large number of pylint warnings
- Removed unused libraries from requirements.txt
- Add .pylintrc with rules to ignore generated protobuf module

### CODE REVIEW

File: src/web/ractl_cmds.py [hunk 1]
```diff
@@ -36,39 +39,39 @@ def get_server_info():
...
-            sahd.append(m)
-        elif mappings[m] == proto.PbDeviceType.SCHD:
-            schd.append(m)
-        elif mappings[m] == proto.PbDeviceType.SCRM:
-            scrm.append(m)
-        elif mappings[m] == proto.PbDeviceType.SCMO:
-            scmo.append(m)
-        elif mappings[m] == proto.PbDeviceType.SCCD:
-            sccd.append(m)
+    for dtype in mappings:
```
Comment (akuker) [inline src/web/ractl_cmds.py:42]: No more single letter variable names! Yay!

File: src/web/templates/index.html [hunk 2]
```diff
@@ -138,75 +138,75 @@
...
+        {% elif file["zip"] %}
             <td>
                 <details>
                     <summary>{{file["name"]}}</summary>
                     <ul>
                     {% for member in file["zip"] %}
-		    <li>
-			<label for="member">{{member}}</label>
-			<form action="/files/unzip" method="post">
+            <li>
```
Comment (akuker) [inline src/web/templates/index.html:149]: @rdmark  - The spacing looks like it got messed up (tabs vs spaces issue?)

File: src/web/web.py [hunk 3]
```diff
@@ -596,90 +671,84 @@ def create_file():
...
 
     process = create_new_image(file_name, file_type, size)
-    if process["status"] == True:
-        flash(f"Drive image created as {file_name}.{file_type}")
-        flash(process["msg"])
-        return redirect(url_for("index"))
-    else:
-        flash(f"Failed to create file {file_name}.{file_type}", "error")
-        flash(process["msg"], "error")
+    if process["status"]:
```
Comment (akuker) [inline src/web/web.py:674]: Are you just assuming that this will always succeed?

File: src/web/ractl_cmds.py (hunk 1 above)
Comment (rdmark) [reply src/web/ractl_cmds.py:42]: The only single-letter variable used now is 'i' for generic iterators (as per pylint default "good-name" rules)

File: src/web/templates/index.html (hunk 2 above)
Comment (rdmark) [reply src/web/templates/index.html:149]: My intention was to standardize on four space indentation across the rascsi-web codebase, and replaced all tabs with four spaces.

It is true that index.html still is a bit of a mess when it comes to indentation though (before and after this change.) I was researching scripts for cleaning up jinja2 template code but haven't found a good one yet. Do you want me to beautify this file as part of this PR?

**Review by akuker — APPROVED** [review]

**Pull request merged**